packages/manager/modules/pci/src/projects/project/storages/containers/containers.service.js

```diff
@@ -53,11 +53,10 @@ export default class PciStoragesContainersService {
 
   getAccessAndToken(projectId, s3StorageType, selectedRegion) {
     if (!s3StorageType) {
-      return this.OvhApiCloudProjectStorage.v6()
-        .access({
-          projectId,
-        })
-        .$promise.then(({ token, endpoints }) => ({
+      return this.$http
+        .post(`/cloud/project/${projectId}/storage/access`)
+        .then(({ data }) => data)
+        .then(({ token, endpoints }) => ({
```

Comment:
This could be simplified
```suggestion
        .then(({ data: { token, endpoints } }) => ({
```

Reply:
it fixed